cli/command/formatter/image.go

```diff
@@ -235,6 +235,11 @@ func (c *imageContext) Digest() string {
 
 func (c *imageContext) CreatedSince() string {
 	createdAt := time.Unix(c.i.Created, 0)
+
+	if createdAt.IsZero() {
+		return "N/A"
```

Comment:
Thanks! Could you change this to return an empty string, so that the `--format '{{ json .}}'` output will show the empty value as-is?

Reply:
Oh man, I'm so sorry, shouldn't do any PRs when I'm waking up hahaha Fixed.

Reply:
LOL no worries! 😂